### PR TITLE
chore: Modify the introduction of Uow usage documentation

### PR DESCRIPTION
**Pull request by zhenlei520**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,12 +118,14 @@ Install-Package MASA.Contrib.Data.Contracts.EF
 ```
 
 ```C#
-builder.Services
-    .AddUoW<CustomDbContext>(dbOptions =>
+builder.Services.AddEventBus(options => {
+    options.UseUoW<CustomDbContext>(dbOptions =>
     {
         dbOptions.UseSqlServer("server=localhost;uid=sa;pwd=P@ssw0rd;database=identity");
         dbOptions.UseSoftDelete(builder.Services);//Start soft delete
```
Comment (doddgu) [inline README.md:125]: I think the comment is needless

File: src/Data/MASA.Contrib.Data.UoW.EF/README.zh-CN.md [hunk 2]
```diff
@@ -10,10 +10,11 @@ Install-Package MASA.Contrib.Data.Contracts.EF
...
-builder.Services
-    .AddUoW<CustomDbContext>(dbOptions =>
+builder.Services.AddEventBus(options => {
+    options.UseUoW<CustomDbContext>(dbOptions =>
     {
         dbOptions.UseSqlServer("server=localhost;uid=sa;pwd=P@ssw0rd;database=identity");
-    })
+    });
+});
 ```
```
Comment (doddgu) [inline src/Data/MASA.Contrib.Data.UoW.EF/README.zh-CN.md:20]: fix `Unchanged files with check annotations`

Comment (zhenlei520) [reply src/Data/MASA.Contrib.Data.UoW.EF/README.zh-CN.md:20]: I don't understand what this means?

**Review by doddgu — COMMENTED** [review]
LGTM

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**